docs/workflows/copy.rst

```diff
@@ -47,47 +47,57 @@ depend on the existence of other content units to be "correct" or meaningful. Fo
+destination.  For example:
 
 * When copying an Advisory (Errata) from one repository to another, all of the RPM packages
   directly referenced by the Advisory will also be copied. (note: this is best-effort, if the
   source repository doesn't contain some of the packages referenced, the copy will proceed
   without them).
-* When copying RPM packages from one repository to another, all of the RPM packages that those
-  packages depend on will also be copied.
+* When copying RPM packages from one repository to another, if dependency-resolution is enabled
+  then all of the RPM packages that those packages depend on are also be copied.
+
+.. note::
 
-The goal is to be as easy to use as possible while maintaining the "correctness" of the repository.
+    In all such cases, Pulp3 engages in a "best effort" attempt to fulfill the requirement. If
+    the source repositor(ies) don't contain all of the referenced content, the copy will proceed
+    without them.
+
+The goal is to be as easy to use as possible, while allowing for a "best effort' at maintaining
+the "correctness" of the repository **when requested to do so**.
 
 Dependency solving
 __________________
 
-When copying RPM packages between repositories, it is likely desirable to ensure that all of the
-RPM dependencies they need to be "installable" on a client system are also present in the
-repository they are being copied to. For example, if you want to add the "hexchat" RPM to a
-new repository, and you want to be able to install it from that repository, the repository should
-also contain its dependencies such as "libnotify" and "gtk3". This applies likewise to all RPM
-concepts that "contain" RPMs also, such as Advisories (Errata), Modules, and PackageGroups. With
-the RPM copy API, you are afforded the option to have all dependencies (and the dependencies of
-those dependencies) copied for you automatically in the background, if they do not already exist
-in the destination repository. In Pulp 2, this feature was called "recursive copy" because in a
-sense it copies dependencies recursively.
+When copying RPM packages, advisories, or modules between repositories, it is useful to ensure
+that all of the RPM dependencies they need to be "installable" on a client system are also present
+in the destination-repository.
+
+For example: if you want to add the "hexchat" RPM to a new repository, and you want to be able
+to install it from that repository, the repository should also contain hexchat's dependencies
+such as "libnotify" and "gtk3".
+
+This applies to all RPM concepts that "contain" RPMs, such as Advisories (Errata), Modules, and
+PackageGroups. With the RPM copy API, you are afforded the option to have all dependencies (and
+the dependencies of those dependencies) copied for you automatically, if they do not already exist
+in the destination repository. In Pulp 2, this feature was called "recursive copy".
```

Comment:
Maybe in parentheses specify that this algorithm was actually called "recursive-conservative".